### PR TITLE
Landing Page Redesign

### PR DESCRIPTION
**Pull request by Favvie**
Modified the existing landing page to the newly designed landing pagex

### CODE REVIEW

File: src/Screens/landingpage/LandingPageScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+
+				<header className="flex lg:hidden justify-between">
+					<Image
+						src={"/images/logoG.svg"}
+						width={150}
+						height={150}
+						alt={"logo"}
+					/>
+					<button>
+						<Image
```
Comment (kcmikee) [inline src/Screens/landingpage/LandingPageScreen.tsx:44]: there isn't any mobile nav

Screenshot 2024-11-09 at 6 00 25 PM

Comment (kcmikee) [reply src/Screens/landingpage/LandingPageScreen.tsx:44]: the icon is there but doesnt do anything

File: src/Screens/landingpage/LandingPageScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+						src="/images/hand-arrow.svg"
+						height={330}
+						width={330}
+						alt="arrow"
+						className="absolute -right-[250px] top-16 hidden lg:block"
+					/>
+				</div>
+
+				<div className="grid lg:grid-cols-2 lg:gap-10 ">
+					<div className="grid lg:grid-cols-2 gap-7">
```
Comment (kcmikee) [inline src/Screens/landingpage/LandingPageScreen.tsx:106]: @Favvie, create a component that will handle each section. 
Incase we want to change something, we don't have to make the change in multiple places

File: src/Screens/landingpage/LandingPageScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+						// className="absolute"
+					/>
+				</div>
+				<div className="flex flex-col justify-center items-center text-center text-guardText gap-5">
+					<h2 className="text-4xl font-semibold">Platform Features</h2>
+					<p className="text-lg font-light">Automated verification, enterprise-grade compliance, and a dedicated demo portal are available.</p>
+				</div>
+
+				<div className="flex flex-col lg:flex-row gap-10 px-5 lg:px-32 items-center">
+					<div className="flex flex-col item-center gap-4 text-white text-center w-full">
```
Comment (kcmikee) [inline src/Screens/landingpage/LandingPageScreen.tsx:325]: component for this too
@Favvie

Comment (kcmikee) [reply src/Screens/landingpage/LandingPageScreen.tsx:325]: @Favvie  
The icons, can you reduce the border width or increase the size of the div (mobile view)

Screenshot 2024-11-09 at 5 56 28 PM

File: src/Screens/landingpage/LandingPageScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,394 @@
...
+								<Image
+									src="/images/play.svg"
+									alt="heroImage"
+									width={25}
+									height={25}
+								/>
+								Schedule a demo
+							</button>
+
+							<button className="bg-primary rounded-full px-8 py-3 w-full lg:w-auto border border-primary">Get started</button>
```
[the remark this answers is not in the document]
Comment (Favvie) [reply src/Screens/landingpage/LandingPageScreen.tsx:77]: done

File: src/Screens/landingpage/LandingPageScreen.tsx (hunk 3 above)
Comment (Favvie) [reply src/Screens/landingpage/LandingPageScreen.tsx]: Alright. Done

File: src/Screens/landingpage/LandingPageScreen.tsx [hunk 5]
```diff
@@ -0,0 +1,365 @@
...
+							alt="heroImage"
+							width={1000}
+							height={768}
+						/>
+					</div>
+				</div>
+			</div>
+
+			<section className="px-5 lg:min-h-96 lg:pt-72 lg:pb-24 lg:pl-32 space-y-28 relative -mt-[100px] lg:mt-0">
+				<div className="space-y-4 max-w-2xl relative">
```
Comment (kcmikee) [inline src/Screens/landingpage/LandingPageScreen.tsx:116]: some text are not showing @Favvie 

Screenshot 2024-11-10 at 7 39 53 PM

Comment (kcmikee) [reply src/Screens/landingpage/LandingPageScreen.tsx:116]: even the buttons. the text are not showing

Comment (Favvie) [reply src/Screens/landingpage/LandingPageScreen.tsx:116]: Yeah. I have fixed it

**Pull request merged**